### ISSUE TITLE
Webpack build errors with CRA and Next.js

### ISSUE THREAD

**Issue by deman4ik** [whawker/react-jsx-highcharts#98]
Tried to install Highstock to latest Creat React App and latest Next.js projects.

Got same error in both projects:
> Module parse failed: Unexpected token (10:19)
> You may need an appropriate loader to handle this file type.
> | class Axis extends Component {
> |
> |   static propTypes = {
> |     dimension: validAxisDimensions.isRequired,
> |     type: validAxisTypes,

What loader do I need?

**Comment by LorenzWieseke** [comment]
Same here, the simple line example works but when I change the import form 
react-jsx-highcharts to react-jsx-highstocks I get the same error. So Highstocks doesn't seem to work for me.

**Comment by deman4ik** [comment]
CRA 1.1.4.
Next.js 6.0.0

**Comment by whawker** [comment]
I mean which version of `react-jsx-highstock`. You can see it if you look in the `package.json` file

**Comment by deman4ik** [comment]
Oh sorry, yes, 2.3.0-alpha.1

**Comment by whawker** [comment]
Damn, I must have published incorrectly, ideally the alpha release shouldn't be downloaded as latest. 

If you `npm install --save react-jsx-highstocks@2.2.1` everything should work fine.



@anajavi, any chance you're available to help me fix errors with the webpack build for highstock?

**Comment by LorenzWieseke** [comment]
npm ERR! 404 Not Found: react-jsx-highstocks@2.2.1

**Comment by LorenzWieseke** [comment]
Also when I get it running now it sais Uncaught Error: Highcharts error #18: www.highcharts.com/errors/18

**Comment by whawker** [comment]
Sorry typo :man_facepalming: 

`npm install --save react-jsx-highstock@2.2.1`

**Comment by LorenzWieseke** [comment]
still getting error above

Highcharts Error #18
The requested axis does not exist
This error happens when you set a series' xAxis or yAxis property to point to an axis that does not exist.

**Comment by anajavi** [comment]
I will take look at it.

**Comment by whawker** [comment]
Are you using [ID props](https://github.com/whawker/react-jsx-highcharts/wiki/id-props-%28important%21%29) for you axis and series?

**Comment by anajavi** [comment]
@whawker: The es build of highstock has references to jsx-highcharts src folder, instead of package.

For example the dist/es/FlagSeries/FlagSeries has: 
```javascript
import Series from 'react-jsx-highcharts/src/components/Series';
```

It can be fixed with importing:
```javascript
import { Series } from 'react-jsx-highcharts';
```

I don't know how it fits your build env though.

**Comment by anajavi** [comment]
@whawker do you want a PR for those changes?

**Comment by whawker** [comment]
Amazing, thanks @anajavi! I don't mind making the changes, thanks so much for your help! Legend!

**Comment by anajavi** [comment]
I think the changes to imports might also require resolve.alias in webpack.config. Something like this (not tested):
```javascript
resolve: {
  alias: {
    "react-jsx-highcharts": 'node_modules/react-jsx-highcharts/src',
  }
}
```

Otherwise the umd-bundle might get both lodash AND lodash-es bundled. I can check this once the changes have landed.

**Comment by whawker** [comment]
@deman4ik, @LorenzWieseke if you `npm install react-jsx-highstock@2.3.0-alpha.2` does it work as expected?

**Comment by whawker** [comment]
@deman4ik, @LorenzWieseke this should be fixed in version 3.0.0, please reopen if it remains an issue.
